### ISSUE TITLE
Cannot find header files on Fedora

### ISSUE THREAD

**Issue by notriddle** [GuillaumeGomez/video-metadata-rs#17]
Fedora stores ffmpeg's header files in `/usr/include/ffmpeg/libav*`, which is not in the default search path.

Consider using [pkg-config](https://crates.io/crates/pkg-config) to set up the include path?

**Comment by GuillaumeGomez** [comment]
Thanks for the feedback. I'll take a look.

**Comment by izgzhen** [comment]
Same problem for OS X 10.11.6

```
#include "..." search starts here:
#include <...> search starts here:
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin/../lib/clang/7.3.0/include
 /Applications/Xcode.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/usr/include
 /Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/MacOSX10.11.sdk/System/Library/Frameworks (framework directory)
```

which doesn't include my header in `/usr/local/include/libav*` (they are mostly s-linked to the brew package of ffmpeg)

**Comment by GuillaumeGomez** [comment]
Should be fixed by #18.
